main.py

```
@@ -47,10 +51,15 @@ def terminate(signalNumber, frame):
         if threading.activeCount() == 1 :
-            logger.debug("Idle")
+            logger.debug("Idle. Sleep 60 sec")
+        else :
+            logger.debug("Sleep 60 sec")
         time.sleep(60)    # pause 60 second
     except  KeyboardInterrupt:
-        logger.critical("Ctrl-X pressed")
+        logger.critical("Program interrupt raised")
+        done = True
+    except Exception:
+        logger.critical("Unexpected error: %s", str(e))
```

Ruff (0.8.2)

`e` may be undefined, or defined from star imports

(F405)